declarative-examples/simple-examples/dockercontainerPort.groovy

```diff
@@ -0,0 +1,60 @@
+pipeline {
+    agent any
+
+    options {
+        timestamps()
+    }
+
+    environment {
+        IMAGE = "custom-tutum"
+    }
+
+    stages {
+        stage('prep') {
+            steps {
+                script {
+                    env.GIT_HASH = sh(
```

Comment:
Is this `GIT_HASH` environment variable required by the script? I cannot see where it is dereferenced.